atuin-client/src/database.rs

```diff
@@ -153,14 +153,15 @@ impl Sqlite {
         Ok(())
     }
 
+    #[allow(clippy::needless_pass_by_value)]
     fn query_history(row: SqliteRow) -> History {
```

Comment:
Clippy suggest to borrow as `&SqliteRow` but this leads to a whole lot of `.map(self.query_history)` fails, where I didn't see the fix.